.golangci.yaml

```diff
@@ -150,5 +154,9 @@ issues:
       path: internal/reconciler/metricpipeline/reconciler_test.go
     - linters: [ dupl ]
       path: main.go
+    - linters: [ dupl ]
+      path: test/e2e/testkit/k8s/verifiers/daemonset.go
+    - linters: [ dupl ]
+      path: test/e2e/testkit/k8s/verifiers/deployment.go
```

Comment:
I think in this case we can actually reuse 80% of the code in both verifiers

Reply:
Instead of duplicating it

Reply:
done